src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts

```diff
@@ -171,7 +171,24 @@ export class NotificationService {
     if (this.hideToasties) {
       return;
     }
-    this.toastr[['error', 'info', 'success'][notification.type]](
+
+    let toastrCall: Function;
+    switch (notification.type) {
+      case NotificationType.error:
+        toastrCall = this.toastr.error;
+        break;
+      case NotificationType.info:
+        toastrCall = this.toastr.info;
+        break;
+      case NotificationType.success:
+        toastrCall = this.toastr.success;
+        break;
+      default:
+        return;
```

Comment:
This doesn't seem code style related, it also makes the code less readable and produces a state that is never be used and if it were used (which should raise an error) no error would be thrown.